packages/openapi-typescript/src/utils.ts

```diff
@@ -55,13 +55,13 @@ export function getSchemaObjectComment(v: CommentObject, indentLv?: number): str
   const output: string[] = [];
 
   // * Not JSDOC tags: [title, format]
-  if (v.title) output.push(`${v.title} `);
-  if (v.summary) output.push(`${v.summary} `);
```

Comment:
I think some of these string escapes mostly came from fields that could contain non-strings, but I think that was really only `example`. I don’t see the harm in removing the string escapes.